crm_phonecall_timesheet/tests/test_crm_phonecall.py

```diff
@@ -31,10 +33,11 @@ def _phonecall_create(self, vals):
         return phonecall
 
     def _test_phonecall_timesheet_asserts(self, phonecall, timesheet):
-        self.assertEqual(phonecall.date[:DATE_LENGTH], timesheet.date)
+        self.assertEqual(
+            phonecall.date.strftime(DEFAULT_SERVER_DATE_FORMAT),
+            timesheet.date.strftime(DEFAULT_SERVER_DATE_FORMAT))
```

Comment:
use `odoo.date_utils`, it's more fun :)

Reply:
date_utils? i will use date objects ```self.assertEqual(phonecall.date.date(), timesheet.date)```

Reply:
Sure, they're datetime objects now. Anyway, is it redundant the `date()` method?

Reply:
And why are you still using `.strftime(DEFAULT_SERVER_DATE_FORMAT)`?

Reply:
phonecall it's an datetime, timesheet it's an 'date'... .date() convert datetime to date.

Reply:
Sure! :)